index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="header_logo">My Logo</div>
+        <div class="header_menu">
+          <img class="menu_icon" src="/images/menu-icon.svg" alt="menu" />
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="headline">
+        <h1 class="headline_title">I’m Mariane Glad to see you!</h1>
+        <p class="headline_content">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <div class="headline_social">
+          <h3 class="social_title">LET’S CONNECT</h3>
+          <ul class="social_links">
+            <li class="social_link">
+              <a href="#"
+                ><img
+                  class="social_icon"
+                  src="/images/twitter.svg"
+                  alt="Twitter"
+              /></a>
+            </li>
+            <li class="social_link">
+              <a href="#"
+                ><img
+                  class="social_icon"
+                  src="/images/linkedin-icon.svg"
+                  alt="Linkedin"
+              /></a>
+            </li>
+            <li class="social_link">
+              <a href="#"
+                ><img class="social_icon" src="/images/github.svg" alt="Github"
+              /></a>
```

Comment:
As you can see from the screenshot below :arrow_down: none of the icons is displaying. I think your paths are not right. Please check your icons paths. I can only see the text rather than the  icons

 :bulb: **Note: When adding a relative path. Please Make sure your start with a dot` .` to target your current directory Your image path should be simailar to src="./images/github.svg" Notice the dot at the beginning**
### Your Design
![connect](https://user-images.githubusercontent.com/26736582/198093053-85eaf278-b81d-49b7-a6c7-5818f07b1807.png)

#### Figma Design

![figma2](https://user-images.githubusercontent.com/26736582/198093180-2504f6d8-7c97-4fff-ae22-1cf45e6ff64d.png)

Reply:
Hi @bienvenuushindi Thanks a lot for your suggestion. But I have a question in here that Should i add more social icons or 3 icon is enough. (I will edit the image path)*